### PR TITLE
WIP - New1 bps 744 create case functional tests

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-744

### Change description ###
Re-created PR-616 with the replica of its branch. PR-616's pipeline is always failing, re-starting pipeline with this new PR.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -55,6 +55,18 @@
     <cve>CVE-2019-16869</cve>
   </suppress>
 
+  <suppress>
```
Comment (nitinprabhu) [inline config/owasp/suppressions.xml]: This is not part of this PR and should go as a separate PR

Comment (nitinprabhu) [reply config/owasp/suppressions.xml]: Also it is recommended to use until date

File: charts/bulk-scan-orchestrator/values.preview.template.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ java:
     DELETE_ENVELOPES_DLQ_MESSAGES_ENABLED: "true"
     DELETE_ENVELOPES_DLQ_MESSAGES_CRON: "0 * * * * *"
     DELETE_ENVELOPES_DLQ_MESSAGES_TTL: "10s"
+    TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```
Comment (nitinprabhu) [inline charts/bulk-scan-orchestrator/values.preview.template.yaml]: Also you will need to bump chart version

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            "Certificate.pdf",
+            "documents/supplementary-evidence.pdf"
+        );
+    }
+
+    @Test
+    public void should_create_case_from_valid_exception_record() throws Exception {
+        // given
+        CaseDetails exceptionRecord = createExceptionRecord("envelopes/new-envelope-with-evidence.json");
+        exceptionRecord.setCreatedDate(null);
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java]: do we need to set this to null ?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/ExceptionRecordCreationTest.java [hunk 4]
```diff
@@ -96,7 +96,8 @@ public void should_create_exception_record_for_new_application() throws Exceptio
         assertThat(caseDetails.getCaseTypeId()).isEqualTo("BULKSCAN_ExceptionRecord");
         assertThat(caseDetails.getJurisdiction()).isEqualTo("BULKSCAN");
 
-        Map<String, String> expectedOcrData = ImmutableMap.of("field1", "value1", "field2", "value2");
+        Map<String, String> expectedOcrData = ImmutableMap.of("first_name", "value1",
```
Comment (nitinprabhu) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/ExceptionRecordCreationTest.java]: may be we can format this ?

**Comment by apereverzin** [comment]
Raised PR-657

**Pull request closed without merging**